### PR TITLE
[kube-prometheus-stack] Bump grafana and kube-state-metrics charts

### PR DESCRIPTION
**Pull request by jkroepke**
#### What this PR does / why we need it

Bump grafana and kube-state-metrics version

#### Which issue this PR fixes

*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*

- fixes #

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Comment by QuentinBisson** [comment]
Any reason why we need a major version bump?

**Comment by jkroepke** [comment]
> Any reason why we need a major version bump?

I though, if a dependency does a major version bump, we also have todo one.

**Comment by QuentinBisson** [comment]
I would think yes in general but this major version bump is actually not impacting here because global.image.registry was changed to global.imageRegistry but global.image.registry  was added fairly recently so users of this chart probably did not even have time to use it :D

**Review by QuentinBisson — APPROVED** [review]

File: charts/kube-prometheus-stack/README.md [hunk 1]
```diff
@@ -82,6 +82,31 @@ _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documen
 
 A major chart version change (like v1.2.3 -> v2.0.0) indicates that there is an incompatible breaking change needing manual actions.
 
+### From 53.x to 54.x
+
+Grafana Helm Chart has bumped to version 7
+
+Please note Grafana Helm Chart [changelog](https://github.com/grafana/helm-charts/tree/main/charts/grafana#to-700).
+
+### From 52.x to 53.x
```
Comment (QuentinBisson) [inline charts/kube-prometheus-stack/README.md:91]: Thanks for the fix

**Pull request merged**